### PR TITLE
Added tutorial for usage with Webpack and bootstrap-sass

### PR DESCRIPTION
**Pull request by drola**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,6 +64,70 @@ Follow me [![twitter](https://img.shields.io/twitter/follow/valorkin.svg?style=s
...
+        loaders: ['css-to-string-loader', 'css-loader', 'resolve-url', 'sass-loader?indentedSyntax=true&sourceMap']
+      },
+      {
+        test: /\.scss$/,
+        loaders: ['css-to-string-loader', 'css-loader', 'resolve-url', 'sass-loader?indentedSyntax=false&sourceMap']
+      },
+      { test: /\.woff(2)?(\?v=[0-9]\.[0-9]\.[0-9])?$/, loader: "url?limit=10000&minetype=application/font-woff" },
+      { test: /\.(ttf|eot|svg)(\?v=[0-9]\.[0-9]\.[0-9])?$/, loader: "file" },
+```
+ - Rename `src/app/app.style.scss` to `src/app/app.style.scss` and add the following line to `src/app/app.styles.scss`:
```
Comment (adrianfaciu) [inline README.md:84]: Probably a small typo here, same file specified twice, probably first should be *.css

Comment (valorkin) [inline README.md:84]: @drola ?

Comment (drola) [inline README.md:84]: Hey, I checked and updated pull request. In absence of indentedSyntax parameter node-sass is smart enough to figure out which syntax to use based on filename.

**Comment by valorkin** [comment]
I am rewriting docs completely, so If you can create getting started webpack + sass md in 
docs\getting-started I will really appreciate it

**Pull request closed without merging**